src/main/java/com/alex/eshop/eshop/RestController/ItemController.java

```diff
@@ -0,0 +1,35 @@
+package com.alex.eshop.eshop.RestController;
+
+import com.alex.eshop.eshop.Entity.Item;
+import com.alex.eshop.eshop.Service.ItemService;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/api")
+public class ItemController {
+    private final ItemService itemService;
+
+    public ItemController(ItemService itemService) {
+        this.itemService = itemService;
+    }
+
+    @GetMapping("/items/last")
+    public List<Item> lastFiveItems() {
+        return itemService.getLastFiveItems();
+    }
+
+    @GetMapping("items/{id}")
+    public Item getItem(@PathVariable Long id) {
+        return itemService.getItemWithCategoryInfo(id);
+    }
+
+    @GetMapping("items/categoryId/{categoryId}")
+    public List<Item> getItemWithCategory(@PathVariable Long categoryId) {
```

Comment:
Use `@RequestParam` instead of `@PathVariable`.
Path must be @GetMapping("/items")